src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -140,4 +141,33 @@ public void getProjectName_ReturnsProjectName() {
         // then
         assertThat(name).isEqualTo(projectName);
     }
+    
+    @Test
+	public void getTagsToExcludeFromChart_defaultConfiguration_ReturnsEmptyList() throws Exception {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        Pattern[] patterns = configuration.getTagsToExcludeFromChart();
+
+        // then
+        assertThat(patterns).isEmpty();
+	}
+    
+    @Test
+	public void getTagsToExcludeFromChart_addPatterns_ReturnsListWithAllPatterns() throws Exception {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        Pattern somePattern = Pattern.compile("");
+        Pattern anotherPattern = Pattern.compile("");
```

Comment:
this test only checks that the configuration can store multiple patterns...
it is irrelevant what is the actual patterns in this case, as we do not check the actual filtering...

i am not sure i understand what is your suggestion to make it more readable?